README.md

```diff
@@ -470,29 +471,29 @@ This option lets you configure how often spawners should be ticked. Higher value
 
 Setting this to `true` replaces the vanilla explosion algorithm with a faster one, at a cost of slight inaccuracy when calculating explosion damage. This is usually not noticeable.
 
-#### enable-treasure-maps
+#### treasure-maps.enabled
 
 `Good starting value: false`
 
 Generating treasure maps is extremely expensive and can hang a server if the structure it's trying to locate is in an ungenerated chunk. It's only safe to enable this if you pregenerated your world and set a vanilla world border.
 
-#### treasure-maps-return-already-discovered
+#### treasure-maps.find-already-discovered
 
 ```
 Good starting values:
-    villager-trade: true
-    loot-tables: true
+      loot-tables: 'true'
```

Comment:
Why is loot-tables value a string where it should be a boolean?

Reply:
That's not important. This is a paper change.
This is not a boolean. It is a string with possible values of `true, false, default`.

If you set `loot-tables: true`, it will automatically change to `loot-tables: 'true'`.

Reply:
well, in that case at least keep it consistent across the board. Either we use the `'` or not in all strings